pynio/instance.py

```diff
@@ -91,8 +91,11 @@ def _get_blocks(self):
         blocks = {}
         for bname, config in self._get('blocks').items():
             btype = config['type']
-            b = blocks_types[btype].copy(bname, instance=self)
-            b.config = config
+            b = deepcopy(blocks_types[btype])
+            b._name = bname
```

Comment:
oh, it can't because that calls "save" which is unnecessary. Should we add a flag to not save blocks?